src/components/MessagesList/MessagesGroup/Message/MessagePart/Mention.vue

```diff
@@ -27,7 +27,8 @@
 			:primary="true" />
 		<NcUserBubble v-else-if="isGroupMention"
 			:display-name="name"
-			:avatar-image="'icon-group-forced-white'" />
+			:avatar-image="'icon-group-forced-white'"
```

Comment:
**Note:** It's better to not bind a static string. In Vue 3 it works faster when it knows that a prop is not being changed.

`avatar-image="icon-group-forced-white"`

Reply:
Ah, nice to know. But I guess we accidentally do that in quite some places (29 occurrences in 15 files)

Reply:
This is kind of a micro-optimization actually. Like "one more comparison during re-rendering".

But there was a rule for this even in Vue 2 for better readability 

https://eslint.vuejs.org/rules/no-useless-v-bind.html#vue-no-useless-v-bind

Reply:
This is kind of a micro-optimization actually. Like "one more comparison during re-rendering".

But there was a rule for this even in Vue 2 for better readability 

https://eslint.vuejs.org/rules/no-useless-v-bind.html#vue-no-useless-v-bind

Reply:
At least now Vue 3 handles it automatically
[proof-link](https://template-explorer.vuejs.org/#eyJzcmMiOiI8ZGl2PlxyXG4gIDxkaXYgZGF0YS10ZXN0PVwic3RyaW5nXCI+RGl2IHdpdGggc3RhdGljIHZhbHVlPC9kaXY+XHJcbiAgPGRpdiA6ZGF0YS10ZXN0PVwiJ2pzLXN0cmluZydcIj5EaXYgd2l0aCBiaW5kaW5nIHN0YXRpYyB2YWx1ZTwvZGl2PlxyXG4gIDxkaXYgOmRhdGEtdGVzdD1cInZhcmlhYmxlXCI+RGl2IHdpdGggYmluZGluZyBkeW5hbWljIHZhbHVlPC9kaXY+XHJcbjwvZGl2PiIsIm9wdGlvbnMiOnt9fQ==)